src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/service/MigrationService.kt

```diff
@@ -18,6 +18,7 @@ abstract class MigrationService<FILTER : Any, NOMIS_ID : Any, MAPPING : Any>(
   private val pageSize: Long,
   private val completeCheckDelaySeconds: Int,
   private val completeCheckCount: Int,
+  private val completeCheckRetrySeconds: Int = 1,
 ) {
```

Comment:
made this configurable but default to old value. Set this to zero speeds up Integration tests